src/pawns.cpp

```diff
@@ -193,7 +191,7 @@ namespace {
             score -= Backward[opposed][f];
 
         if (connected)
-            score += Connected[opposed][phalanx][more_than_one(supported)][relative_rank(Us, s)];
+            score += Connected[opposed][!!phalanx][more_than_one(supported)][relative_rank(Us, s)];
```

Comment:
probably a matter of personal preference, but I find `phalanx != 0` more readable than `!!phalanx`.